app/forms/network-interface-edit.tsx

```diff
@@ -65,6 +84,69 @@ export function EditNetworkInterfaceForm({
     >
       <NameField name="name" control={form.control} />
       <DescriptionField name="description" control={form.control} />
+      <FormDivider />
+
+      <div className="flex flex-col gap-3">
+        {/* We have to blow this up instead of using TextField for better layout control of field and ClearAndAddButtons */}
+        <div>
+          <FieldLabel id="transitIps-label" htmlFor="transitIp" optional>
+            Transit IPs
+          </FieldLabel>
+          <TextInputHint id="transitIps-help-text" className="mb-2">
```

Comment:
Is this ID unused? Should it be referenced in a `aria-describedby` or whatever? Looks like this magic only happens `tooltipText` is truthy, which seems kinda weird tbh

https://github.com/oxidecomputer/console/blob/7daaa337c1895b68ea414e7d059656da2b4fe3d1/app/components/form/fields/TextField.tsx#L132

Reply:
Fixed in [aee425a](https://github.com/oxidecomputer/console/pull/2437/commits/aee425abd25b745d87006371c84ba1a07fb112c1)